custom/motioncontrollers/BangBangController.java

```diff
@@ -18,6 +19,7 @@
 	protected double A;
 	protected double F;
 	protected double threshold;
+	protected static final String DEFAULT_SMARTDASHBOARD_PREFIX = "!!";
```

Comment:
If this actually works, I think it is great. I would watch out for improper nonalphanumeric character parsing in NetworkTables though.